src/main/java/com/es/phoneshop/model/cart/Cart.java

```diff
@@ -0,0 +1,30 @@
+package com.es.phoneshop.model.cart;
+
+import com.es.phoneshop.model.product.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cart {
+    private List<CartItem> items;
+
+    public Cart() {
+        this.items = new ArrayList<>();
+    }
+
+    public List<CartItem> getItems() {
+        return items;
+    }
+
+    public CartItem findItemInCart(Product product) {
```

Comment:
findItem is better name of method, cause class is named Cart, no need to duplicate